### ISSUE TITLE
Bandwidth of transducer

### ISSUE THREAD

**Issue by AliKafaei** [clariusdev/cast#126]
Hi, 
We have L15-HD transducer. We would like to know what is the bandwidth of this transducer. In the manual, it said that the transducer frequency range is 5 - 15 MHz. Does the bandwidth depend on the settings we select? 
Can we change the bandwidth ?
Regards,
Ali

**Comment by clariusk** [comment]
Yes, correct, 5-15, there are no options to change bandwidth, Clarius optimizes each preset and depth setting for transmit and receive frequencies.

**Comment by AliKafaei** [comment]
So when you say optimizing, does that mean it changes the bandwidth or it is always fixed?

**Comment by clariusk** [comment]
It is fixed, but the frequency slides as a function of depth

**Comment by AliKafaei** [comment]
Would you please clarify more the frequency slides?

**Comment by clariusk** [comment]
so if we transmit at say 10 MHz, we may adjust the receive filtering from 15MHz in the near field, and linearly slide it to the imaging depth to say 8MHz, these are configured per preset and depth range

**Comment by AliKafaei** [comment]
Does the sliding info per depth is also saved so that we know the bandwidth for each depth ?

**Comment by clariusk** [comment]
if you have specific presets and probe models, we can give you that info

**Comment by AliKafaei** [comment]
Thanks, we will contact you when we decide about the specific presets we are going to use

**Comment by AliKafaei** [comment]
Hi, 
We use two specific settings:
1-  MSK, maximum depth = 4 cm
2-  RF data, maximum depth = 4 cm
I am not sure that MSK preset also can provide RF data or not.

**Comment by clariusk** [comment]
Transmit of 14MHz
Receive from 15MHz to 10MHz over 4cm
RF does not go through demodulation, therefore no RX applied

**Comment by AliKafaei** [comment]
Sorry, by RX, do you mean Recieve filtering ?

**Comment by AliKafaei** [comment]
Hi,
We have done an experiment to verify the spectrum of RF data. We put a metal plate in a container filled with distilled water. We get the received RF data and took FFT (we considered that the sampling frequency is 60 Mhz as it was mentioned in the yml file).
This is the spectrum :
![image](https://github.com/clariusdev/cast/assets/36259911/d3021ab0-1b45-4771-986f-4b101efe97e1)
This is the Bmode 
![image](https://github.com/clariusdev/cast/assets/36259911/e98cd3ab-5063-4ed1-b41f-d98ad9b32f4e)
This is from yml file 
![image](https://github.com/clariusdev/cast/assets/36259911/1e48f498-d77a-4f9f-af51-0acb84070382)
The spectrum does not match with the specified center frequency in the yml file.

**Comment by AliKafaei** [comment]
Hi,
Just a kind reminder about the bandwidth probe. We are going to collect data soon.
Regards,

**Comment by clariusk** [comment]
thanks for the graph and data, we're not too sure, as our acoustic measurements show probes range from 9.5-10.5mhz center frequency.

**Comment by AliKafaei** [comment]
Is it possible that the sampling frequency is different than 60 (the value mentioned)? 
I used this value to generate the graphs.

**Comment by AliKafaei** [comment]
We tested Clarius using two modes, research and MSK,  and calculated the spectrum using multi-taper method.
![image (33)](https://github.com/clariusdev/cast/assets/36259911/ee500276-8153-4ec7-85dd-2641e0cb413c)

![image (34)](https://github.com/clariusdev/cast/assets/36259911/0a7033b7-e458-450d-8788-ee1b52e020a7)

According to yml file the center frequency of MSK and research modes are 14 Mhz and 10 Mhz (cannot be verified in the figures), respectively. 
We also would like to know why there are contents outside of bandwidth in research mode.
